lib/pengine/utils.c

```diff
@@ -12,7 +12,6 @@
 #include <crm/msg_xml.h>
 #include <crm/common/xml.h>
 #include <crm/common/util.h>
-#include <crm/common/output.h>
```

Comment:
`#include <crm/common/output.h>` was added in the previous commit. By removing it, utils.c becomes untouched by this PR.